### PR TITLE
Readme and man pages regenerated to reflect updates

### LINKED ISSUE #1003: Docs: Some short links to sections in README.md were broken when headings were changed
tip: if you add a space and emoji links will need a trailing `-` adding

Fixed by #997

### PR DESCRIPTION
**Pull request by philclifford**
docs: Readme and man pages rebuilt

Besides capturing the manual changes to Readme.md this also has regenerated and thus aligned the current output of `usage()` and the supported distros.

A couple of changes had also broken short links to headings, which have been found and fixed thanks to `markdownlint`

Closes #1003 

Co-authored-by: frafra [frafra@users.noreply.github.com](mailto:frafra@users.noreply.github.com)

### CODE REVIEW

**Comment by zen0bit** [comment]
As you are documentation master here @philclifford
Can pandoc also generate some diagram how quickget and quickemu internaly works?
(spoted something)
that would be very helpfull..

File: README.md
Comment (zen0bit) [inline README.md:1]: line 995/1028: - <https://github.com/acidanthera/OpenCorePkg/tree/master/Utilities/macrecovery> was removed

Comment (philclifford) [reply README.md:1]: It was still useful for many versions so deserves recognition ...

Comment (zen0bit) [reply README.md:1]: ok

**Comment by philclifford** [comment]
@zen0bit Not natively in the way I think you mean.  

I think it would be necessary to add information and/or use some tooling to generate diagrams from the code. Having been a fan of CWEB and Z my first thoughts naturally turn to 
http://www.literateprogramming.com/tools.html ( mostly not the droids you are seeking but ... ) 

Coincidentally I was reminded of this thread today :  https://twitter.com/thingskatedid/status/1386077306381242371

For me the preferred options are always to find the path of least work and maximum automation/self-generation potential, with tooling that can build a workflow with minimal intervention and minimal need for instrumentation work. Or find an IDE with the visualisations and workflows built in...

Don't forget that markdown comes with extensions in most environments ...
 
```mermaid
sequenceDiagram
    participant Alice
    participant Bob
    Alice->>John: Hello John, how are you?
    loop Healthcheck
        John->>John: Fight against hypochondria
    end
    Note right of John: Rational thoughts <br/>prevail!
    John-->>Alice: Great!
    John->>Bob: How about you?
    Bob-->>John: Jolly good!
```

There are tools that can build a function diagram by parsing the code, but whilst that's easy for `quickemu` they tend to get upset by the complexity of `quickget` and fail miserably.

**Comment by zen0bit** [comment]
Thanks for answer

**Comment by zen0bit** [comment]
If is posible for quickemu, must be doable for quickget also.

just leave one os per edge case. Should make quickget smaller enough.

like
leave there just:
- alma
- alpine
- macos
- ubuntu
- xubuntu
- windows
...

that should be enough for basic flowchart.
??

**Comment by philclifford** [comment]
> If is posible for quickemu, must be doable for quickget also.

Except quickget code is mutated by parameters .. a static analyser or parser looking for paths through the code has to cope with run-time-generated code paths ...

**Comment by philclifford** [comment]
``` shell
echo "digraph {
  delete_disk -> delete_shortcut
  delete_vm -> delete_shortcut
  vm_boot -> ignore_msrs_alert
  vm_boot -> get_port
  vm_boot -> enable_usb_passthrough
  vm_boot -> check_cpu_flag
  vm_boot -> efi_vars
  vm_boot -> is_numeric
  Main -> ignore_msrs_always
  Main -> delete_disk
  Main -> delete_vm
  Main -> snapshot_apply
  Main -> snapshot_create
  Main -> snapshot_delete
  Main -> snapshot_info
  Main -> vm_boot
  Main -> start_viewer
  Main -> shortcut_create
  Main -> usage
  Main -> display_param_check
  Main -> sound_card_param_check
  Main -> viewer_param_check
  Main -> parse_ports_from_file
  Main -> is_numeric
  Main -> monitor_send_cmd
}
" |  dot -Tsvg > theGraph.svg && eom theGraph.svg
```

**Comment by zen0bit** [comment]
What is dot?

**Comment by philclifford** [comment]
``` shell
dpkg -S $(command -v dot)
graphviz: /usr/bin/dot
```
one of a "rich set of graph drawing tools" beloved of greybeards

**Comment by zen0bit** [comment]
> ```shell
> echo "digraph {
> ```

I cam play with that.
Thanks for info

**Review by zen0bit — APPROVED** [review]

**Review by flexiondotorg — APPROVED** [review]

**Pull request merged**